test/controllers/works_controller_test.rb

```diff
@@ -1,116 +1,468 @@
 require 'test_helper'
 
 describe WorksController do
+  let (:movie) {works(:movie)}
+
   describe "root" do
     it "succeeds with all media types" do
       # Precondition: there is at least one media of each category
+      get root_path
 
+      must_respond_with :success
     end
 
     it "succeeds with one media type absent" do
       # Precondition: there is at least one media in two of the categories
+      movie.category = "book"
```

Comment:
You're not removing movies in one category here.

You should be destroying all works with the `book` category.